### DISCUSSION TITLE
[BUG] Add-PnPTermToTerm with Sites.Selected permissions not working

### DISCUSSION THREAD

**Discussion by Richter79** [pnp/powershell#3046] [General]
### Reporting an Issue or Missing Feature
Granted access to an Azure App via Grant-/Set-PnPAzureADAppSitePermissions with FullControl (Sites.Selected permissions)
It does not seem to be possible to create terms in a SiteCollection termset via Add-PnPTermToTerm.
Permission for TermStore.ReadWrite.All is not allowed, since global termstore manipulations must not allowed for the App.

### Expected behavior 
Term can be created.

### Actual behavior 
Add-PnPTermToTerm: The current user has insufficient permissions to perform this operation.
![image](https://user-images.githubusercontent.com/19924238/233387662-69a936a7-0b94-421e-a31d-a3c9073e5662.png)
New-PnPList is working, Get-PnPTerm is working.

### Steps to reproduce behavior
- Create an Azure App, granted Selected.Sites
![image](https://user-images.githubusercontent.com/19924238/233391631-226ba2e9-492a-44a6-b6f4-f231dd589704.png)
- create / upload certificate
- Grant-PnPAzureADAppSitePermissions (only read and write is possible)
- Get-PnPAzureADAppSitePermission
- Set-PnPAzureADAppSitePermission -PermissionId $permissionsId -Permissions FullControl
- Connect-PnPOnline -Url URL -ClientId APPID -Thumbprint CERTTHUMBPRINT -Tenant TENANTID

### What is the version of the Cmdlet module you are running?
PnP.PowerShell 2.1.1

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
You will need to provide TermStore permissions, otherwise this cmdlet will not work.

**Reply by Richter79** [reply]
Thanks. There is no other possibility? Since we are not allowed to get global termstore permissions. Therefore the terms are stored in the local term store.

**Reply by anilmaskara-cell** [reply]
Are you able to make this work..in similar situation
